stumpy/aamp_stimp.py

```diff
@@ -320,13 +324,21 @@ def PAN_(self):
         """
         Get the transformed (i.e., normalized, contrasted, binarized, and repeated) pan
         matrix profile
+
+        Parameters
+        ----------
+        None
         """
         return self.pan().astype(np.float64)
 
     @property
     def M_(self):
         """
         Get all of the (breadth first searched (level) ordered) subsequence window sizes
+
+        Parameters
+        ----------
+        None
```

Comment:
The section for `Returns`  is missing.

Reply:
Since it is a class method, we do not require a return

Reply:
I see. Good to know :) 
Btw, its sibling, `stimp`, has the section `Returns`. e.g.: function `M_`.

However, being inconsistent may not be a big deal here. I mean we can try to be perfect but the docstring is good and clear.